.github/workflows/build_workflow.yml

```diff
@@ -21,6 +21,6 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Build with Maven
-        run: mvn clean install --no-transfer-progress
+        run: mvn clean install
```

Comment:
Any reason why you need to remove `--no-transfer-progress` ?

Reply:
to be honest, I don't remember. Since this is draft, maybe it was just to try something. I've check latest build logs, and I don't see any "noise".

I'll close it.